lib/components/molecules/QuoteMedia/QuoteMedia.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+/**
+ *
+ * QuoteMedia
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import type { Node } from 'react';
+import type { Props } from './types';
+
+import Heading from '../../atoms/Heading/Heading';
+
+import Blockquote from '../../atoms/Blockquote/Blockquote';
+
+import styles from './QuoteMedia.style';
+
+const QuoteMedia = ({ className, title, author, quoteContent, children }: Props): Node => {
+  return (
+    <div className={className}>
+      <Heading HeadingType="h2" className="title">
+        {title}
+      </Heading>
+      <Blockquote className="blockquote">
+        <p className="blockquote__content">{quoteContent}</p>
```

Comment:
<p className="blockquote__content">{quoteContent}</p>
        <footer className="blockquote__author">{author}</footer>

Should be part of the Blockquote component because this component will not accept anything else